src/main/java/Duke.java

```diff
@@ -1,10 +1,37 @@
-public class Duke {
+import java.util.Scanner;
+import java.util.Arrays;
+import java.util.ArrayList;
+public class Duke{
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        String dash = "__________________________________";
+        String greet = dash + "\nHello I'm Duke\nWhat can I do for you?\n" + dash;
+        System.out.println(greet);
+
+        ToDoList toDo = new ToDoList();
+
+        while (true) {
+            Scanner scan = new Scanner(System.in);
+            String input = scan.nextLine();
+            String[] Temp = input.split(" ");
+            if(Temp[0].equals("mark")|Temp[0].equals("unmark")){
+                int numberToMark = Integer.parseInt(Temp[1]);
```

Comment:
The naming can be better. In this case, since it's the index of the task, something like taskIndex might offer better readability.